### ISSUE TITLE
Make a Navbar

### ISSUE THREAD

**Issue by ACM-CIET** [ACM-CIET/ACM_Hacking-Wizards#6]
**Try to add a responsive navbar and add useful links of ACM CIET**

**Comment by vansh-attri** [comment]
Hi, I would like to work on this. Can you please assign it to me?

**Comment by ACM-CIET** [comment]
Yes @vansh-attri really go ahead and feel free to contact us for any help, and sorry for belated reply!

**Comment by vansh-attri** [comment]
@ACM-CIET No problem

**Comment by vansh-attri** [comment]
Made a pull requset #20 for the same. Please check
<img width="936" alt="Screenshot 2023-10-13 202126" src="https://github.com/ACM-CIET/ACM_Hacking-Wizards/assets/138808830/e59dafad-89c6-476b-8670-ee33fbf6e681">

**Comment by ACM-CIET** [comment]
Great work @vansh-attri just please sync your fork so that we can merge it without conflicts
